fairlearn/postprocessing/_interpolated_thresholder.py

```diff
@@ -41,12 +42,35 @@ class InterpolatedThresholder(BaseEstimator, MetaEstimatorMixin):
         between 0 and 1.
     prefit : bool
         if `True` then the base estimator is not fitted in :meth:`fit`.
+    predict_method : {'auto', 'predict_proba', 'decision_function', 'predict'\
+            }, default='auto'
+        Defines which method of the ``estimator`` is used to get the output
+        values.
+        - 'auto': use one of ``predict_proba``, ``decision_function``, or
+          ``predict``, in that order.
+        - 'predict_proba': use the second column from the putput of
+          `predict_proba`. It is assumed that the second column represents the
+          positive outcome.
+        - 'decision_function': use the raw values given by the
+          `decision_function`.
+        - 'predict': use the hard values reported by the `predict` method.
+          This is equivalent to what is done in [1]_.
+
+        .. versionadded:: 0.7
```

Comment:
Ooooh I love this! Should we open an issue to add this to every method everywhere?

Follow-up question: does this only work for methods or classes as well?

Reply:
Yes, that'd be a nice sprint issue.